### PR TITLE
[l10n]> New pot + French 

### PR DESCRIPTION
**Pull request by fxbenard**
Ready for translate.wordpress.org import. French will hit 100% there - Ready for language pack

### CODE REVIEW

**Comment by sumobi** [comment]
Thanks @fxbenard! What was the reasoning for removing the `../` in the .pot file? Shouldn't they correctly show where each string is?

File: languages/easy-image-gallery.pot [hunk 1]
```diff
@@ -1,64 +1,86 @@
...
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Poedit-SearchPath-0: ..\n"
+"X-Generator: Poedit 1.8.5\n"
+"X-Poedit-Basepath: ..\n"
+"X-Poedit-WPHeader: easy-image-gallery.php\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"X-Poedit-KeywordsList: __;_e;_n:1,2;_x:1,2c;_ex:1,2c;_nx:4c,1,2;esc_attr__;"
+"esc_attr_e;esc_attr_x:1,2c;esc_html__;esc_html_e;esc_html_x:1,2c;_n_noop:1,2;"
+"_nx_noop:3c,1,2;__ngettext_noop:1,2\n"
+"X-Poedit-SearchPath-0: .\n"
```
Comment (fxbenard) [inline languages/easy-image-gallery.pot]: @sumobi SearchPath is still here

File: languages/easy-image-gallery.pot [hunk 2]
```diff
@@ -1,64 +1,86 @@
...
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 1.6.3\n"
-"X-Poedit-KeywordsList: __;_e\n"
-"X-Poedit-Basepath: .\n"
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Poedit-SearchPath-0: ..\n"
+"X-Generator: Poedit 1.8.5\n"
+"X-Poedit-Basepath: ..\n"
```
Comment (fxbenard) [inline languages/easy-image-gallery.pot]: @sumobi and BasePath here

**Comment by fxbenard** [comment]
@sumobi they're still here, your Poedit version is so old that it doesn't show the same order. Last version grabs the description in header as well. If you add the Text Domain: of course there

**Comment by fxbenard** [comment]
And anyway the pot will be of no use when you hit translate.wordpress ;)

**Comment by fxbenard** [comment]
That's it i got it.
BasePath and SearchPath are now inverted, and that is much better.
The base is the root of the plugin so ../ back from the languages folder. 
And the strings are in the SearchPath so ./

I updated the pot from Poedit, it worked like a charm.
I used the pot to create the French po, it worked like a charm.

**Comment by sumobi** [comment]
:+1: excellent, thanks for the explanation

**Pull request merged**